### PR TITLE
Tighten typing in eclcompress/API

### PR DESCRIPTION
**Pull request by berland**

### CODE REVIEW

File: src/subscript/eclcompress/eclcompress.py [hunk 1]
```diff
@@ -190,20 +188,22 @@ def acceptedvalue(valuestring: str) -> bool:
...
-        keywordsets (list): List of 2-tuples, (start, end) indices in
+        keywordsets: List of 2-tuples, (start, end) indices in
             line number in the deck, referring to individual sections
             of distinct keywords.
-        filelines (list): list of lines (strings) from Eclipse deck, cleaned.
+        filelines: list of lines from Eclipse deck, cleaned.
 
     Returns:
-        list:  List of strings to be used as a replacement Eclipse deck
+        Strings to be used as a replacement Eclipse deck
```
Comment (asnyv) [inline src/subscript/eclcompress/eclcompress.py:206]: In another method you have switched to e.g. `List of strings [...]` and `List of 2-tuples [...]` , so maybe change this for consistency, even though the type hinting is even more descriptive?
```suggestion
        List of strings to be used as a replacement Eclipse deck
```

**Review by asnyv — CHANGES_REQUESTED** [review]

File: src/subscript/eclcompress/eclcompress.py (hunk 1 above)
Comment (berland) [reply src/subscript/eclcompress/eclcompress.py:206]: The intention was not to say "list of something" when the type hinting already says the same (and it is included next to the text in sphinx-generated docs), only a textual description, and then "strings" in plural is the textual description of "List[str]"

Comment (berland) [reply src/subscript/eclcompress/eclcompress.py:206]: Consistency now attempted.

**Review by asnyv — APPROVED** [review]
⛱

**Pull request merged**